### ISSUE TITLE
Fatal C error in a basic `coeval` run

### ISSUE THREAD

**Issue by lingyuanji** [21cmfast/21cmFAST#204]
A fatal C error is encountered when executing a very basic `coeval` run.

A minimal reproduction:
```
$ conda create --name 21cmfast python=3
$ conda activate 21cmfast
$ conda install -c conda-forge 21cmFAST
$ python -c 'import py21cmfast as p21c; p21c.run_coeval(redshift=8.0)'
```

Error messages:
```
/Users/LingyuanJi/anaconda3/envs/21cmfast/lib/python3.9/site-packages/py21cmfast/_cfg.py:49: UserWarning: Your configuration file is out of date. Updating...
  warnings.warn(
/Users/LingyuanJi/anaconda3/envs/21cmfast/lib/python3.9/site-packages/py21cmfast/_cfg.py:32: UserWarning: Your configuration file is out of date. Updating...
  warnings.warn("Your configuration file is out of date. Updating...")
2021-02-02 19:23:40 | ERROR   | heating_helper_progs.c | T_RECFAST:204 [pid=63380] | T_RECFAST: Unable to open file: /Users/LingyuanJi/.21cmfast/External_tables/recfast_LCDM.dat for reading.
2021-02-02 19:23:40,123 | ERROR | _utils.py::_process_exitcode() | In function: ComputeIonizedBox.  Arguments: [...]
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/LingyuanJi/anaconda3/envs/21cmfast/lib/python3.9/site-packages/py21cmfast/wrapper.py", line 2318, in run_coeval
    ib2 = ionize_box(
  File "/Users/LingyuanJi/anaconda3/envs/21cmfast/lib/python3.9/site-packages/py21cmfast/wrapper.py", line 1620, in ionize_box
    return box.compute(
  File "/Users/LingyuanJi/anaconda3/envs/21cmfast/lib/python3.9/site-packages/py21cmfast/_utils.py", line 938, in compute
    _process_exitcode(exitcode, self._c_compute_function, args)
  File "/Users/LingyuanJi/anaconda3/envs/21cmfast/lib/python3.9/site-packages/py21cmfast/_utils.py", line 53, in _process_exitcode
    raise FatalCError
py21cmfast._utils.FatalCError: 21cmFAST is exiting.
```
In `[...]`, there is a very long list of arguments that I omit to make the error message readable. I could show it if this turns out to be relevant. My `~/.21cmfast` is empty.

System information:
```
macOS 11.1
Python 3.9.1
conda 4.9.2
```

**Comment by BradGreig** [comment]
Hi @lingyuanji, this looks like an existing known issue (#190). 

For the moment it seems that the temporary solution is to download the source code directly, and then manually copy the contents of the `user_data` folder to your created directory (`/Users/LingyuanJi/.21cmfast/`).

Hopefully that should resolve your issue. Please let us know if it still doesn't work for you.

**Comment by lingyuanji** [comment]
Thank you for mentioning #190. The solution you provided works perfectly!

Why the code won't work with an empty `~/.21cmfast`? Also, why is the file content not copied correctly to `~/.21cmfast` during `conda install`?

**Comment by BradGreig** [comment]
Hi @lingyuanji, there appears to be a bug in the `conda install` which is preventing the copying of the files. We are hoping to have this sorted soon.

As to why it won't work with an empty `~/.21cmfast`. The missing files are pre-computed data that is required to be read in by the code. Thus, in the absence of the requisite file, it will fail.

**Comment by steven-murray** [comment]
Note this will hopefully be fixed by #205. Closing this now, as it is a duplicate of #190.
